### ISSUE TITLE
"connu pour être un état pétrolier"

### ISSUE THREAD

**Issue by GPPassos** [UniversalDependencies/UD_French#2]
I have some doubts about this particular construction in this sentence:

```
# sent_id = fr-ud-train_00944
# text = Anzoátegui, connu pour être un état pétrolier, produit également du gaz.
1	Anzoátegui	Anzoátegui	PROPN	_	_	10	nsubj	_	SpaceAfter=No
2	,	,	PUNCT	_	_	1	punct	_	_
3	connu	connaître	VERB	_	Gender=Masc|Number=Sing|Tense=Past|VerbForm=Part	1	acl	_	_
4	pour	pour	ADP	_	_	5	mark	_	_
5	être	être	VERB	_	VerbForm=Inf	3	acl	_	_
6	un	un	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	7	det	_	_
7	état	état	NOUN	_	Gender=Masc|Number=Sing	5	obj	_	_
8	pétrolier	pétrolier	ADJ	_	Gender=Masc|Number=Sing	7	amod	_	SpaceAfter=No
```

It's fine that `acl(Anzoátegui,connu)`, but there are some points:

1. `acl(connu,être)` seems very strange, as "_connu_" isn't a noun (or nominal).
2. "_être un état pétrolier_" would usually have "_état_" as head, as it's an equational non-verbal predication. I know this is a harder construction, but why should it be different here?

**Comment by dan-zeman** [comment]
My knowledge of French is extremely basic, so I may be missing something, but:

I would analyze it as

cop(état, être)
xcomp(connu, état)

The copula part seems pretty straightforward. "To be a state..." is just the infinitive version of "it is a state", i.e. a nonverbal predicate with copula.

The xcomp relation is because of secondary predication. (An alternative would be attaching it to the noun, i.e. acl(Anzoátegui, état), but I think that this is obligatory depictive, see http://universaldependencies.org/u/overview/complex-syntax.html#secondary-predicates.)

**Comment by jnivre** [comment]
The fact that "être" and not "état" is the head of the infinitival phrases looks like a plain error to me. It should definitely be:

cop(état, être)

as Dan says. When it comes to the relation between "connu" and "état", I would suggest "advcl". The  phrase "pour être un état pétrolier" modifies the predicate "connu" and is introduced by the marker "pour". I don't think this is a case of secondary predication.

**Comment by GPPassos** [comment]
Indeed, I agree with the `advcl` analysis.
(Although my knowledge of French is basic as well)

**Comment by mcdm** [comment]
Yes Joakim is right. This is an error coming from the automatic conversion of the Google treebank which didn't use copula as head.

**Comment by perrier54** [comment]
I agree with Dan about the dependency from "connu" to "état". The only possible label is XCOMP because in French, "connu" governs the complement "pour être un état pétrolier"; therefore ADVCL is not possible. Between CCOMP and XCOMP, we must take XCOMP because the subject of "connu" is necessarily the subject of the infinitive introduced by "pour".
